style.css

```
@@ -53,3 +53,29 @@ main {
   grid-area: 10/ 5 / span 3 / span 2;
   background-color: whitesmoke;
 }
+
+
+/* suggested changes*/
+.welcome {
+  grid-area: 1/ 1 / 1 / 20;
+}
+.header-grid {
+  grid-area: 1/1/1/15;
```

grid-area : row-start/column-start/row-end/column-end; //this defines the grid area, and spans automatically between defined start and end row columns.
```suggestion
  grid-area: 1/1/2/20; // can be changed as follows to set as (starts from first row first column and spreads upto 2nd row and 20th column)
```